src/main/java/io/appform/dropwizard/actors/actor/BaseActor.java

```diff
@@ -106,6 +113,13 @@ protected boolean handle(Message message, MessageMetadata messageMetadata) throw
         return handle(message);
     }
 
+    /*
+        Override this method in your code in case you want to handle the expired messages separately
+     */
+    protected boolean handleExpiredMessages(Message message, MessageMetadata messageMetadata) throws Exception {
```

Comment:
What does the return value indicate ? Who would use this return value ?

Reply:
This return value is the acknowledgment for the rmq.

If a client wants to handle the expired messages differently, let's say sending an event and if that fails wants to retry then can send false and the retry strategy would kick in.

Reply:
Ack